ClientApp/KachnaOnline/src/app/users/user-profile/user-profile.component.ts

```diff
@@ -0,0 +1,29 @@
+// user-profile.component.ts
+// Author: David Chocholatý
+
+import { Component, OnInit } from '@angular/core';
+import {AuthenticationService} from "../../shared/services/authentication.service";
+import {NgForm} from "@angular/forms";
+
+@Component({
+  selector: 'app-user',
+  templateUrl: './user-profile.component.html',
+  styleUrls: ['./user-profile.component.css']
+})
+export class UserProfileComponent implements OnInit {
+  public isCardInfoCollapsed: boolean = false;
+
+  constructor(
+    public authenticationService: AuthenticationService,
+  ) { }
+
+  gamificationConsentTooltipText:string = "Tohle je souhlas s gamifikací."
+  cardNumberTooltipText:string = "Tohle je krásná karta."
```

Comment:
Again, `cardCode` would be better than `cardNumber`

Reply:
Good point. Fixing.